astronomer/airflow/version_check/templates/update-available.html

```diff
@@ -1,27 +1,43 @@
-{% if current_user.is_authenticated and cea_update_available %}
-  {% set level = "error" if cea_update_available.level == "critical" else "warning" %}
-  <div class="alert alert-{{ level }} ac-update-notice">
-    {% if "can_dismiss" | is_item_visible("UpdateAvailable") %}
-      <button data-href="{{ url_for('UpdateAvailable.dismiss', version=cea_update_available.version) }}" title="Ignore this update, and hide this message from all users">&times; Ignore this update</button>
+{% if current_user.is_authenticated %}
+  {% if cea_update_available %}
+    {% set level = "error" if cea_update_available.level == "critical" else "warning" %}
+    <div class="alert alert-{{ level }} ac-update-notice">
+      {% if "can_dismiss" | is_item_visible("UpdateAvailable") %}
+        <button data-href="{{ url_for('UpdateAvailable.dismiss', version=cea_update_available.version) }}" title="Ignore this update, and hide this message from all users">&times; Ignore this update</button>
+      {% endif %}
+      <p><strong>A new version of {{ cea_update_available.app_name }} is available.</strong></p>
+      <p>
+        {% if cea_update_available.url %}
+          <a href="{{ cea_update_available.url }}">
+        {% endif %}
+          Version {{ cea_update_available.version }}
+        {% if cea_update_available.url %}
+          </a>
+        {% endif %}
+        was released on
+        <time datetime="{{ cea_update_available.date_released.isoformat() }}">{{ cea_update_available.date_released.strftime("%Y-%m-%d") }}</time>
+        {%- if cea_update_available.level == "critical" %}
+          and this is a critical fix
+        {%- endif -%}
+        .
+      </p>
+      {% if cea_update_available.description %}
+        <p>{{ cea_update_available.description }}</p>
+      {% endif %}
+    </div>
+  {% endif %}
+
+  {% if cea_eol_notice %}
+    {% set eol_level = "warning" if cea_eol_notice.days_to_eol > 0 else "error" %}
+    {% if not cea_eol_notice.dismissed_until or current_time > cea_eol_notice.dismissed_until %}
```

Comment:
This feels like it's in the wrong spot. Can't we do this on the python side of things, like we do for the update notice (filter in the db query)?

Reply:
Changed it